openapi/components/schemas/global/about.yaml

```diff
@@ -102,7 +109,7 @@ properties:
         type: boolean
       commitHash:
         type: string
-      usersCount:
+      userCount:
```

Comment:
some people just want to see the world burn

Reply:
hahaha, that was actually the response, I fixed the spec, not changed it :joy: